tests/test_bridge_domain.rb

```diff
@@ -24,6 +24,11 @@ class TestBridgeDomain < CiscoTestCase
   @@cleaned = false # rubocop:disable Style/ClassVars
 
   def cleanup
+    Vlan.vlans.each do |vlan, obj|
+      # skip reserved vlans
+      next if vlan == '1'
+      obj.destroy
+    end
```

Comment:
Moved the code to common location and also created another remove_all_bridge_domains section for bds